embark-ui/src/reducers/index.js

```diff
@@ -369,6 +369,13 @@ function editorTabs(state = [], action) {
   return state;
 }
 
+function commandHistory(state = [], action) {
+  if (action.type === COMMAND_HISTORY[SUCCESS]) {
+    return action.commandHistory;
+  }
+  return state;
+}
```

Comment:
This could have been an entity, no? Or we want to move away from entities because they are too confusing?

Reply:
That couldn't, entity manage object with id (or fake id), in this case each time I do a request I want to replace everything.